src/transformStateWithClones.js

```diff
@@ -65,7 +65,45 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
-}
+  const states = [];
+  let stateClone = {
+    ...state,
+  };
```

Comment:
```suggestion
  let stateClone = { ...state };
```
You can write it in one line but it's ok